cryodrgn/models.py

```diff
@@ -1,17 +1,22 @@
 """Pytorch models"""
 
+from typing import Optional, Tuple, Type
```

Comment:
@zhonge told me a bunch of these models can be deleted. Now would be a good time... @zhonge can you comment?